dingo/bindings/bindings.cpp

```diff
@@ -120,13 +125,23 @@ double HPolytopeCPP::apply_sampling(int walk_len,
    } else if (method == 5) { // dikin walk {
       uniform_sampling<DikinWalk>(rand_points, HP, rng, walk_len, number_of_points,
                                   starting_point, number_of_points_to_burn);
-   } else if (method == 6) { // dikin walk {
+   } else if (method == 6) { // john walk {
```

Comment:
what is changed here?

Reply:
changed the comments (instead of dikin walks, they are john and vaidya walks), just for clarity.

Reply:
ah, it was a typo, thanks!